clients/admin-ui/src/theme/ant.ts

```diff
@@ -1,49 +1,65 @@
 import { AntThemeConfig } from "fidesui";
+import palette from "fidesui/src/palette/palette.module.scss";
+
+/**
+ * Order of priority for styling
+ * 1. Ant Design default theme
+ * 2. FidesUI palette colors
+ * 3. Ant Design custom theme (this file, which also includes some custom override colors not found in the palette)
+ * 4. global CSS variables (as a last resort when styling Ant components, should rely on the palette vars)
+ * 5. tailwindcss (for layout and spacing only)
+ * 6. SCSS modules (for custom-component-specific styles)
+ */
 
 export const antTheme: AntThemeConfig = {
+  cssVar: true,
   token: {
-    colorPrimary: "#2b2e35",
-    colorInfo: "#2b2e35",
-    colorSuccess: "#5a9a68",
-    colorWarning: "#e59d47",
-    colorError: "#d9534f",
-    colorLink: "#2272ce",
-    colorBgBase: "#ffffff",
+    colorPrimary: palette.FIDESUI_MINOS,
+    colorInfo: palette.FIDESUI_MINOS,
+    colorSuccess: palette.FIDESUI_SUCCESS,
+    colorWarning: palette.FIDESUI_WARNING,
+    colorError: palette.FIDESUI_ERROR,
+    colorLink: palette.LINK,
+    colorBgBase: palette.FIDESUI_FULL_WHITE,
     borderRadius: 4,
     wireframe: true,
-    colorTextBase: "#2b2e35",
-    colorErrorBg: "#ffdcd6",
-    colorErrorBorder: "#f2aca5",
-    colorWarningBg: "#ffecc9",
-    colorWarningBorder: "#ffdba1",
-    colorSuccessBorder: "#5a9a68",
-    colorPrimaryBg: "#e3e0d9",
-    colorBorder: "#e6e6e8",
+    colorTextBase: palette.FIDESUI_MINOS,
+    colorErrorBg: "#ffdcd6", // custom override
+    colorErrorBorder: "#f2aca5", // custom override
+    colorWarningBg: "#ffecc9", // custom override
+    colorWarningBorder: "#ffdba1", // custom override
+    colorSuccessBorder: palette.FIDESUI_SUCCESS,
+    colorPrimaryBg: palette.FIDESUI_SANDSTONE,
+    colorBorder: palette.FIDESUI_NEUTRAL_100,
+    zIndexPopupBase: 1500, // supersede Chakra's modal z-index
```

Comment:
Does this bump up all other zIndexes based on this? I didn't even know they had that, much simpler than overriding component by component.

Reply:
This is specific to the popup component used in the Select. It really came up while I was working on the Select migration but included it in this PR anyway.